src/lib/helloworld/main_module.rb

```diff
@@ -0,0 +1,78 @@
+# frozen_string_literal: true
+
+# License stuff
+require 'yast'
+
+Yast.import 'UI'
+Yast.import 'Label'
+
+# Simple Hello World! fully functioning example
+# @note This module can be changed using the YaST development tool
+#   "create-new-package"
+module HelloWorld
```

Comment:
to follow conventions it should be Y2HelloWorld and same for path `y2hello_world`

Reply:
Updated in commit.